doc: add the v1.2.0-patch1 ISO for the usb installation issue (#435)
harvester/docs: docs/install/usb-install.md

:::caution
**Known Issue:** For the v1.2.0 ISO image, there is a known [issue](https://github.com/harvester/harvester/issues/4510) where the interactive ISO installation will get stuck using the USB method.

> Now you can use the patchd [ISO](https://releases.rancher.com/harvester/v1.2.0/harvester-v1.2.0-patch1-amd64.iso) to resolve this issue. We only repack v1.2.0 ISO with the correct lable, and there is no other change. Also, there is a sha512 [file](https://releases.rancher.com/harvester/v1.2.0/harvester-v1.2.0-patch1-amd64.iso.sha512sum) for verify the ISO.

Reviewer: ```suggestion
Now you can use the patchd [ISO](https://releases.rancher.com/harvester/v1.2.0/harvester-v1.2.0-patch1-amd64.iso) to resolve this issue. We only repack v1.2.0 ISO with the correct lable, and there is no other change. Also, you can use this sha512 [file](https://releases.rancher.com/harvester/v1.2.0/harvester-v1.2.0-patch1-amd64.iso.sha512sum) to verify the ISO.
```

Author: Hi @vickyhella, thanks for the review!
Updated

> Now you can use the patchd [ISO](https://releases.rancher.com/harvester/v1.2.0/harvester-v1.2.0-patch1-amd64.iso) to resolve this issue. We only repack v1.2.0 ISO with the correct lable, and there is no other change. Also, you can use this sha512 [file](https://releases.rancher.com/harvester/v1.2.0/harvester-v1.2.0-patch1-amd64.iso.sha512sum) to verify the ISO.

Reviewer: ```suggestion
Now you can use the patchd [ISO](https://releases.rancher.com/harvester/v1.2.0/harvester-v1.2.0-patch1-amd64.iso) to resolve this issue. We only repack the v1.2.0 ISO with the correct label and nothing else is changed. Also, you can use this sha512 [file](https://releases.rancher.com/harvester/v1.2.0/harvester-v1.2.0-patch1-amd64.iso.sha512sum) to verify the ISO.
```

Reviewer: One more suggestion... (sorry for missing this one earlier)

Author: Updated, thanks!

> Now you can use the patchd [ISO](https://releases.rancher.com/harvester/v1.2.0/harvester-v1.2.0-patch1-amd64.iso) to resolve this issue. We only repack the v1.2.0 ISO with the correct label and nothing else is changed. Also, you can use this sha512 [file](https://releases.rancher.com/harvester/v1.2.0/harvester-v1.2.0-patch1-amd64.iso.sha512sum) to verify the ISO.

Reviewer 2: ```suggestion
To resolve this, you can use the patched [ISO](https://releases.rancher.com/harvester/v1.2.0/harvester-v1.2.0-patch1-amd64.iso). This patched version only corrects the label; no other changes have been made. You can also use the related sha512 [file](https://releases.rancher.com/harvester/v1.2.0/harvester-v1.2.0-patch1-amd64.iso.sha512sum) to verify the ISO.
```
